### DISCUSSION TITLE
Which package supports Vue 2.7?

### DISCUSSION THREAD

**Discussion by kingyue737** [intlify/vue-i18n#1036] [Q&A]
We have `vue-i18n-composable` and `vue-i18n-bridge`. Do they support Vue 2.7 which has built-in Composition API ?

**Comment by kingyue737** [comment]
`vue-i18n-composable` now supports Vue 2.7

Update: 
`vue-i18n-bridge` v9.2.0 also supports Vue 2.7 now

**Comment by 6XGate** [comment]
`vue-i18n-bridge` support for 2.7 isn't implemented correctly thought. The [`@vue/composition-api` README](https://github.com/vuejs/composition-api#readme) states the library will only support Vue 2.6 and earlier. `vue-demi` should be the source for any imported offical Vue composition APIs.
